src/main/java/duke/task/Events.java

```diff
@@ -0,0 +1,64 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents tasks of type events, inherit from Task abstract class.
+ * Events have a description, date, start time and end time.
+ */
+public class Events extends Task {
+    protected LocalDate localDate;
+    protected LocalTime localStartTime;
+    protected LocalTime localEndTime;
+
+    /**
+     * Constructor of the Events class.
+     *
+     * @param description description of an event in String.
+     * @param localDate date when the event happens.
+     *                  Must be in the format yyyy-mm-dd.
```

Comment:
Should this be how multiline Javadoc comments are indented? I am not very certain myself.